nidirect_common/src/InvalidateTaxonomyListCacheTags.php

```diff
@@ -50,4 +72,20 @@ public function invalidateForEntity(EntityInterface $entity) {
     }
   }
 
+  /**
+   * Get parent taxonomy term for landing pages.
+   */
+  private function checkLandingPageParent($entity, $tid) {
+    // Landing page is a special case.
+    if ($entity->type->target_id == 'landing_page') {
+      // In this case we want to invalidate the cache tag for
+      // the parent of the current term.
+      $term = \Drupal::entityTypeManager()->getStorage('taxonomy_term')->load($tid);
```

Comment:
You'll want to use your classes entityTypeManager variable here rather than the static service.

Reply:
Silly me ! All done now ....